projects/grass-addons/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mkdir build
+cd build
+cmake ..
```

Comment:
It doesn't work, there is no CMakeLists.txt in the project :slightly_smiling_face:

Reply:
Hii @tyler92 i had updated build.sh script but still build check is failing Could you please help in this to create build.sh script so it will pass testcases 
Thanks